### DISCUSSION TITLE
Command cooldowns for specific arguments

### DISCUSSION THREAD

**Discussion by DeathSmurf** [EssentialsX/Essentials#4922] [Ideas and Feature Suggestions] [answered]
I need to add cooldown to /home bed, without adding cooldown to /home.

the problem that the way to teleport to bed is with the /home command.

How can I make only the /home bed command start cooling and not all /home commands?

WHAT I WANT (cooldown):
**cooldown 60min = " /home bed "**  
(config.yml --->_command-cooldowns_:----->bed: 3600 ) <this does not work :c>

**cooldown 1min = " /home (others)" (home: 60)**
(config.yml --->_command-cooldowns_:----->home: 60 )

Sorry, I don't write very well in English.

**Comment by mdcfe** [comment, marked as answer]
You can use a regular expression to match any `/home` commands that *aren't* `/home bed`. I haven't tested this but it should work:
```
command-cooldowns:
  'home bed': 3600
  '^^home(?! bed)( .*)?': 60
```

For more explanation, see the [wiki page on command cooldowns](https://essentialsx.net/wiki/Command-Cooldowns.html). For help with writing and testing regular expressions, you can use a tool like [Regex101](https://regex101.com) or [Regexr](https://regexr.com).
